blueocean-dashboard/src/main/js/creation/bitbucket/server/BbAddServerDialog.jsx

```diff
@@ -0,0 +1,155 @@
+import React, { PropTypes } from 'react';
+import { observer } from 'mobx-react';
+import debounce from 'lodash.debounce';
+import { Dialog, FormElement, TextInput } from '@jenkins-cd/design-language';
+
+import ServerErrorRenderer from './ServerErrorRenderer';
+
+
+let t = null;
```

Comment:
@vivek seems weird to put it in the module scope. Better to assign as an instance on `BBAddServerDialog`?